lib/tokenizers/tokenizer.ex

```diff
@@ -183,43 +170,33 @@ defmodule Tokenizers.Tokenizer do
     Base.encode32(etag, case: :lower, padding: false)
   end
 
-  @typedoc """
-  Options to set on the loaded tokenizer.
-
-    * `:additional_special_tokens - a list of special tokens to append to the tokenizer.
-      Defaults to `[]`.
-
-    * `:padding` - Override for padding configuration. Currently the only supported
-      value is `:none` to disable padding. By default the configuration is restored
-      from the file.
-
-    * `:truncation` - Override for truncation configuration. Currently the only supported
-      value is `:none` to disable truncation. By default the configuration is restored
-      from the file.
-
-  """
-  @type load_options ::
-          [
-            additional_special_tokens: [String.t() | Tokenizers.AddedToken.t()],
-            padding: :none,
-            truncation: :none
-          ]
-
   @doc """
   Instantiate a new tokenizer from the file at the given path.
   """
-  @spec from_file(path :: String.t(), load_options()) :: {:ok, t()} | {:error, term()}
-  defdelegate from_file(path, options \\ []),
-    to: Tokenizers.Native,
-    as: :tokenizer_from_file
+  @spec from_file(path :: String.t(), keyword()) :: {:ok, t()} | {:error, term()}
+  def from_file(path, options \\ []) do
+    if Keyword.has_key?(options, :additional_special_tokens) do
+      IO.warn(
+        "passing :additional_special_tokens as an option is deprecated. Use add_special_tokens/2 instead"
+      )
+    end
```

Comment:
Now that we have `add_special_tokens/2`, I don't think it makes sense to support both long term.